src/Poseidon/CLI/Serve.hs

```diff
@@ -85,11 +86,12 @@ parseArchiveConfigFile :: (MonadIO m) => FilePath -> m ArchiveConfig
 parseArchiveConfigFile = decodeFileThrow
 
 data ArchiveSpec = ArchiveSpec
-    { _archSpecName        :: ArchiveName
-    , _archSpecPaths       :: [FilePath]
-    , _archSpecDescription :: Maybe String
-    , _archSpecURL         :: Maybe String
-    , _archSpecDataURL     :: Maybe String
+    { _archSpecName               :: ArchiveName
+    , _archSpecPaths              :: [FilePath]
+    , _archSpecDescription        :: Maybe String
+    , _archSpecURL                :: Maybe String
+    , _archSpecDataURL            :: Maybe String
+    , _archSpecExcludePacsFromMap :: Maybe [String]
```

Comment:
I'm aware that `Maybe [String]` is ugly, but I think it's necessary to play nicely with aeson (see [here](https://stackoverflow.com/questions/21345365/how-to-omit-empty-lists-using-aeson-derivejson)).

Reply:
I've encountered this before and solved it with a simple one-liner (like [here](https://github.com/poseidon-framework/poseidon-analysis-hs/blob/0e9e2e079b509254f5f3cb1cde08515634d12116/src/Poseidon/Analysis/FStatsConfig.hs#L81)